src/main.rs

```diff
@@ -98,8 +107,9 @@ fn update_dependencies(
     cargo_toml_path: &Path,
     crates_versions: &BTreeMap<String, String>,
     overwrite: bool,
+    update: bool,
```

Comment:
Also, maybe we could rename this to `check_only` since the function is already called `update_dependency`, so it is kind of implied that it would update the dependencies.

Reply:
Updated the flag name!